platform/linuxbsd/detect.py

```diff
@@ -119,8 +119,16 @@ def configure(env):
     if env["bits"] == "default":
         env["bits"] = "64" if is64 else "32"
 
-    if env["arch"] == "" and platform.machine() == "riscv64":
-        env["arch"] = "rv64"
+    machines = {
+        "riscv64": "rv64",
+        "ppc64le": "ppc64le",
+        "ppc64": "ppc64",
+        "ppcle": "ppcle",
+        "ppc": "ppc",
```

Comment:
`arch` is something we use internally (specifically in the places where you had to parse it to disable modules). It doesn't seem necessarily to have this much granularity in the architecture variants, at least until we actually need and want to do separate configuration for `ppc64` vs `ppc` or `ppc64le` vs `ppc64`.

Similarly, we only have `armv7` and `armv8` (or `arm64` depending on the platform) and not a full blown `armv7l armv7hl armv7hnl armv8l armv8hl armv8hnl armv8hcnl`.

Reply:
alright, i guess having just `ppc` is okay for now then?

Reply:
or maybe `ppc` and `ppc64`, since this affects the final executable name

Reply:
I guess `ppc` and `ppc64` makes sense yeah.